tensorflow/python/keras/benchmarks/BUILD

```diff
@@ -74,25 +74,33 @@ cuda_py_test(
     ],
 )
 
+py_library(
+    name = "benchmark_util",
+    srcs = ["benchmark_util.py"],
+    deps = [
+        "//tensorflow:tensorflow_py",
+        "//third_party/py/numpy",
+    ],
+)
+
```

Comment:
I think it's not a problem, since we have `keras_exmaples_benchmark_test` and I deleted, so I placed the `benchmark_util` in different lines and it shows green. You can check with [there](https://github.com/tensorflow/tensorflow/pull/41249/files#diff-186433d6a449914056aee3a2fc32e63cL91) and I already updated, we can see if `windows check` is work or not.

Reply:
Ah, I see. It's deleted at the bottom. LGTM.